ui/src/app/views/confirm-playground-mode-modal.tsx

```diff
@@ -0,0 +1,68 @@
+import * as Cookies from 'js-cookie';
+import * as React from 'react';
+
+import {Button} from 'app/components/buttons';
+import {CheckBox} from 'app/components/inputs';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import colors from 'app/styles/colors';
+
+export interface Props {
+  onCancel: Function;
+  onContinue: Function;
+}
+
+interface State {
+  checked: boolean;
+}
+
+export class ConfirmPlaygroundModeModal extends React.Component<Props, State> {
+
+  public static DO_NOT_SHOW_AGAIN = 'DO_NOT_SHOW_AGAIN_CONFIRM_PLAYGROUND_MODE_MODAL';
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      checked:  Cookies.get(ConfirmPlaygroundModeModal.DO_NOT_SHOW_AGAIN) === String(true)
+    };
+  }
+
+  toggleChecked() {
+    const newState = !this.state.checked;
+    this.setState({checked: newState});
+    Cookies.set(ConfirmPlaygroundModeModal.DO_NOT_SHOW_AGAIN, String(newState));
+  }
+
+  render() {
+    return <Modal onRequestClose={() => { this.props.onCancel(); }}>
+      <ModalTitle>Playground Mode</ModalTitle>
+      <ModalBody>
+        <p style={{color: colors.primary}}>
+          Playground mode allows you to explore, change and run the code,
+          but your edits will not be saved.
+        </p>
+        <p style={{color: colors.primary}}>
+          To save your work, choose <b>Make a Copy</b> from the <b>File Menu</b>
```

Comment:
Nitpick: based on the PR screenshot, you may need an &nbsp; after the </b> to add a space after File Menu and before the next word.

Reply:
good catch